jenkins/config.xml

```diff
@@ -137,7 +137,7 @@
 					</envVars>
 				</org.csanchez.jenkins.plugins.kubernetes.PodTemplate>
 			</templates>
-			<serverUrl>https://172.28.128.4:8443</serverUrl>
+			<serverUrl>https://kubernetes</serverUrl>
```

Comment:
Is this service available in all namespaces? I know it works with `kubernetes.default.svc` but didn't know if it was available as `kubernetes` only in non-`default` namespace?